manual/vcredist2005/vcredist2005.nuspec

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <version>8.0.50727.619501</version>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/manual/vcredist2005</packageSourceUrl>
+    <authors>Microsoft</authors>
+    <owners>chocolatey-community,MarkRobertJohnson</owners>
+    <licenseUrl>http://msdn.microsoft.com/en-US/cc300389.aspx</licenseUrl>
+    <projectUrl>https://www.microsoft.com/en-us/download/details.aspx?id=26347</projectUrl>
+    <id>vcredist2005</id>
+    <title>Microsoft Visual C++ 2005 Redistributable Package</title>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+The Microsoft Visual C++ 2005 Redistributable Package (x86 &amp; x64) installs runtime components of Visual C++ Libraries required to run applications developed with Visual C++ on a computer that does not have Visual C++ 2005 installed. 
+
+## Notes
+
+- This will install **both the 32 and 64 bit versions** on a 64 bit OS. The 32 bit version will only be installed on a 32 bit OS.
+- [Latest supported Visual C++ downloads](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads)
+</description>
+    <summary>This package installs runtime components of C Runtime (CRT), Standard C++, ATL, MFC, OpenMP and MSDIA libraries. For libraries that support side-by-side deployment model (CRT, SCL, ATL, MFC, OpenMP) t</summary>
+    <releaseNotes>
+    - MFC Security Update: A security issue has been identified leading to a vulnerability in MFC applications that are built with Visual Studio 2010 and ship the Microsoft Visual C++ 2010 Service Pack 1 Redistributable Package.
+    - KB Article: [KB2538242](https://support.microsoft.com/kb/2538242)
+    - Security bulletin: [MS11-025](https://technet.microsoft.com/en-us/security/Bulletin/MS11-025)
+    - Microsoft published: 8/9/2011
+    </releaseNotes>
+    <copyright>https://www.microsoft.com/en-us/legal/intellectualproperty/copyright</copyright>
+    <tags>visual c++ redistributable 2005 studio admin</tags>
```

Comment:
Please also include the `vcredist2005` in the tag, as well as 1 of the obligatory tags listed here: https://github.com/chocolatey-community/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#122-obligatory-tags (`freeware` sounds appropriate in this case)